src/script/src/python/vector.rs

```diff
@@ -1045,7 +1051,13 @@ pub mod tests {
     /// test the paired `val_to_obj` and `pyobj_to_val` func
     #[test]
     fn test_val2pyobj2val() {
-        rustpython_vm::Interpreter::without_stdlib(Default::default()).enter(|vm| {
+        rustpython_vm::Interpreter::with_init(Default::default(), |vm| {
```

Comment:
Can we remove the stdlib? I found that it's impossible in old version.

Reply:
> Can we remove the stdlib? I found that it's impossible in old version.

So we need to remove freeze-stdlib and only allow user to use a limited `greptime_builtin` module instead?